l10n/en/firefox/nightly/whatsnew.ftl

```diff
@@ -16,6 +16,13 @@ nightly-whatsnew-every-4-to-5-weeks = Every 4 to 5 weeks, a new major version of
 
 nightly-whatsnew-this-is-a-good = This is a good time to thank you for helping us make { -brand-name-firefox } better and to give you some pointers to documentation, communication channels and news sites related to { -brand-name-nightly } that may be of interest to you.
 
+# Variables:
+#   $blog (url) - link to https://blog.nightly.mozilla.org/
+#   $mastodon (url) - link to https://mozilla.social/@FirefoxNightly
+#   $twitter (url) - link to https://twitter.com/FirefoxNightly
+nightly-whatsnew-if-you-want-to-v3 = If you want to know what’s happening around { -brand-name-nightly } and its community, reading our <a href="{ $blog }">blog</a> and following us on <a href="{ $mastodon }">Mastodon</a> or <a href="{ $x }">X</a> are good starting points!
```

Comment:
The X link here is 404 since the variable is called `$x` here but `twitter` in the template